src/shared/components/palette/case-flag/components/manage-case-flags/manage-case-flags.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="form-group" [formGroup]="formGroup">
+  <div class="govuk-form-group" [ngClass]="{'form-group-error': errorMessages.length > 0}">
+    <fieldset class="govuk-fieldset" aria-describedby="manage-case-flag-heading">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+        <h1 id="manage-case-flag" class="govuk-fieldset__heading">
```

Comment:
@tarikdogan-hmcts @waynepHMCTS - the `id` needs to be `"manage-case-flag-heading"`, so it matches the `aria-describedby` attribute